lib/dependencies/RequireEnsureDependency.js

```diff
@@ -2,24 +2,30 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var NullDependency = require("./NullDependency");
-var DepBlockHelpers = require("./DepBlockHelpers");
+"use strict";
+const NullDependency = require("./NullDependency");
+const DepBlockHelpers = require("./DepBlockHelpers");
 
-function RequireEnsureDependency(block) {
-	NullDependency.call(this);
-	this.block = block;
-}
-module.exports = RequireEnsureDependency;
+class RequireEnsureDependency extends NullDependency {
+	constructor(block) {
+		super();
+		this.block = block;
+	}
 
-RequireEnsureDependency.prototype = Object.create(NullDependency.prototype);
-RequireEnsureDependency.prototype.constructor = RequireEnsureDependency;
-RequireEnsureDependency.prototype.type = "require.ensure";
+	get type() {
+		return "require.ensure";
+	}
+}
 
-RequireEnsureDependency.Template = function RequireEnsureDependencyTemplate() {};
+RequireEnsureDependency.Template = class RequireEnsureDependencyTemplate {
+	apply(dep, source, outputOptions, requestShortener) {
+		const depBlock = dep.block;
+		const wrapper = DepBlockHelpers.getLoadDepBlockWrapper(depBlock, outputOptions, requestShortener, "require.ensure");
+		const startBlock = wrapper[0] + "(";
```

Comment:
Id say its fine as is. @Kovensky would you open an issue for using prefer-template